.gitlab-ci.yml

```diff
@@ -8,6 +8,7 @@ variables:
   GITLABCI_VERSION:    "2.1"
   GITLABCI_IMAGE:      "gitlabci"
   CONTRIBUTOR_IMAGE:   "domjudge/domjudge-contributor"
```

Comment:
Can we change this to have a (dummy) value and description? See https://docs.gitlab.com/ee/ci/yaml/#prefill-variables-in-manual-pipelines
As value I’d suggest not actually using a version but something like “enter version”

Reply:
I choose the semantic versioning format, but I agree this is better.